action.yml

```
@@ -0,0 +1,71 @@
+name: 'Composite Action'
+description: 'This workflow is able to build and push a specified Image from Tutor to Docker Hub.'
+
+inputs:
+  python-version:
+    description: 'This input refers the version of Python. It is useful since venv will be created from it.'
+    required: false
+    default: '3.8'
+  tutor-version:
+    description: 'This input refers the version of Tutor to be installed. It takes 15.3.0 by default.'
+    required: true
+    default: '15.3.0'
+  repository-name:
+    description: 'This input refers the name of the repository already created in Docker Hub.'
+    required: true
+  image-to-be-built:
+    description: 'This input refers the image to be built. It takes openedx by default.'
+    required: true
+    default: openedx
```

I think there should not be default at all, it must be always specified,  mustn't it?.
```suggestion
  image-to-be-built:
    description: 'Image to be built.'
    required: true
```
@sergivalero20

---

Yes, your'e right!